cms/views_test.py

```diff
@@ -42,7 +42,6 @@ def test_home_page_view(client):
         f'<a id="actionButton" class="btn btn-primary text-uppercase px-5 py-2 action-button" href="#">Watch Now</a>'
         not in content
     )
-    assert "dropdown-menu" in content
```

Comment:
why do we need to remove it ?

Reply:
This was introduced mistakenly in a previous PR by @umarmughal824 and the original check was `assert not "dropdown-menu" in content`. Consequently this was failing tests so I removed it. The original assertion is still there a little later in the test.

Reply:
I could not understand it.

Reply:
That was a side-effect of another bugfix. Removed in this one.